StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory.swift

```diff
@@ -213,7 +213,7 @@ class PaymentSheetFormFactory {
     func make() -> PaymentMethodElement {
         switch paymentMethod {
         case .instantDebits, .linkCardBrand:
-            return makeInstantDebits()
+            return makeInstantDebits(countries: configuration.billingDetailsCollectionConfiguration.allowedCountriesArray)
```

Comment:
Is this correct? if it is empty -- should we default to the previous behavior, or... should we not even show instantDebits/linkCardBand ?

Reply:
Same behavior as for cards: If `allowedCountries` is empty, then `allowedCountriesArray` is nil and we don't filter. As soon as the merchant passes in any value to `allowedCountries`, we only accept those.